pkg/controller/dataimportcron-controller_test.go

```diff
@@ -738,6 +739,7 @@ var _ = Describe("All DataImportCron Tests", func() {
 				Expect(labels[cc.LabelDefaultPreference]).Should(Equal(cc.LabelDefaultPreference))
 				Expect(labels).Should(ContainElement(cc.LabelDefaultPreferenceKind))
 				Expect(labels[cc.LabelDefaultPreferenceKind]).Should(Equal(cc.LabelDefaultPreferenceKind))
+				Expect(labels).Should(HaveKeyWithValue(cc.LabelDynamicCredentialSupport, "true"))
```

Comment:
Unrelated to the overall PR but I'd appreciate it if you could replace the above with `HaveKeyWithValue` assertions and maybe rename the func to `ExpectLabels`.

Reply:
Done